apps/dashboard/src/app/knowledge-box/knowledge-box-home/knowledge-box-home.component.html

```diff
@@ -269,104 +269,124 @@
           </nsi-home-container>
         }
 
-        @if ((isKbContrib | async) && (isAccountManager | async)) {
-          <nsi-home-container class="usage-charts">
-            <div class="title-m title-with-action">
-              <div class="title-with-icon">
-                <pa-icon name="chart"></pa-icon>
-                {{ 'dashboard-home.usage.title' | translate }}
+        @if (isKbContrib | async) {
+          @if (isRemiMetricsEnabled | async) {
+            <nsi-home-container>
+              <div class="title-with-action">
+                <div class="title-with-icon title-m">
+                  <pa-icon name="heart"></pa-icon>
+                  {{ 'dashboard-home.health-status.title' | translate }}
+                  <small class="body-s">({{ 'kb.metrics.period.last-7d' | translate }})</small>
+                </div>
+                <a
```

Comment:
A minor thing, it seems that this link is visible by "writer" users , but hey are not allowed to access the metrics page.

Reply:
oh, good catch! thanks

Reply:
fixed